src/Microsoft.Azure.WebJobs.Extension.Dapr/DaprExtensionConfigProvider.cs

```diff
@@ -67,9 +68,7 @@ public void Initialize(ExtensionConfigContext context)
             stateRule.BindToInput<JObject>(daprStateConverter);
             stateRule.BindToInput<Stream>(daprStateConverter);
             stateRule.BindToInput<byte[]>(daprStateConverter);
-
-            // TODO: This does not work for nulls and value types. Need a better way of doing this conversion.
-            stateRule.BindToInput<object?>(daprStateConverter);
+            stateRule.BindToInput<OpenType>(typeof(DaprStateGenericsConverter<>), this.daprClient);
```

Comment:
This makes me wonder, should other converters also use a Generic Open type instead of byte[] and Newtonsoft objects?

Reply:
I am keeping specific converter for string, stream, daprstaterecord, as they don't need specific conversion, so having Newtonsoft library won't be adding any help with generic converter. In case of byte[], we we return empty array byte when it is null. So, keeping it as separate converter.

However, removed JToken and JObject converter to use generic converter.